ios/ble/Utility/TransferHandler.swift

```diff
@@ -186,7 +190,7 @@ extension TransferHandler: PeripheralCommunicatorProtocol {
             } else if status == 1 {
                 EventEmitter.sharedInstance.emitNearbyMessage(event: "send-vc:response", data: "\"REJECTED\"")
             }
-            Wallet.shared.destroyConnection(isSelfDisconnect: true)
+            destroyConnection?()
```

Comment:
I think this should be calling `handleDestroyConnection` method, not `destroyConnection` method. Cross check with Android implementation in `onNotificationReceived` method in `Wallet.kt` file.

Also this is the confusion we get into if we don't differentiate method names based on what it does. From method names `handleDestroyConnection` and `destroyConnection` sounds same. But both are doing opposite activities as per the current implementation.

Reply:
This destroyConnection is a callback from transferHandler to wallet which actually invokes wallet handleDestroyConnection.

Reply:
Okay.